hub/modules/inittopic/inittopic.bal

```diff
@@ -0,0 +1,23 @@
+import ballerina/jballerina.java;
```

Comment:
Can we rename this to something else inittopic is not sounding good

Reply:
we can call this internal_topic_helper, if this is ok lets change the method names, error message also to match

Reply:
done

Reply:
As discussed lets change kafka health check also to be underscore separated.

Reply:
resolved